README.md

```diff
@@ -4,7 +4,7 @@
 
 A basic GitHub repository example for new Call for Code projects and those that join the Call for Code with The Linux Foundation deployment initiative. Not all sections or files are required. You can make this as simple or as in-depth as you need.
 
-_Read this in other languages: [English](README.md), [한국어](README.ko.md), [português](README.pt_br.md)._
+_Read this in other languages: [English](README.md), [한국어](./docs/README.ko.md), [português](./docs/README.pt_br.md)._
```

Comment:
Just need to make sure we rebase on latest in `main` with the updated buttons.

Reply:
The latest changes from the `main` branch are merged